src/Microsoft.ML.Data/Transforms/ValueToKeyMappingTransformer.cs

```diff
@@ -811,6 +811,16 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
                     var terms = GetTermsAndIds<double>(iinfo, out termIds);
                     node.AddAttribute("keys_floats", terms);
                 }
+                else if (info.TypeSrc.GetItemType().Equals(NumberDataViewType.Int64))
```

Comment:
I think this is different enough that it warrants some test. Can you please add one?